ext-selenium/src/main/scala/teststate/selenium/TestStateExt.scala

```diff
@@ -0,0 +1,27 @@
+package teststate.selenium
+
+import org.openqa.selenium.WebDriver
+import teststate.dsl.Dsl
+import teststate.typeclass.ExecutionModel
+
+object TestStateExt {
+
+  final class DslExt[F[_], R, O, S, E](private val dsl: Dsl[F, R, O, S, E]) extends AnyVal {
+
+    def withSeleniumTab(tab: R => Tab[WebDriver])(implicit EM: ExecutionModel[F]): Dsl[F, R, O, S, E] =
+      dsl.withActionMod(_.mod(actionDef => ros =>
+        actionDef(ros).map(actionFn => () =>
+          EM.flatten(EM.point(tab(ros.ref).use(_ =>
+            actionFn()))))))
```

Comment:
Wrong. Only correct wher `F = Id`. Add `Tab#useF` or more methods to `ExecutionModel`